notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    cat_pre = OrdinalEncoder(
</pre><p>I'm not that familiar with sklearn pipelines but if you added something like <code>SimpleImputer</code> to the categorical columns, then <code>cat_pre</code> could be a pipeline that you pass into <code>ColumnTransformer</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>